### DISCUSSION TITLE
Vermeidung zeitgleicher Eingaben

### DISCUSSION THREAD

**Discussion by Outi77** [OpenLigaDB/OpenLigaDB-Samples#92] [Ideas]
Servus zusammen,

mir ist es in letzter Zeit immer wieder mal vorgekommen, dass ich beim Eintragen der Ergebnisse (vielmehr Torschütze und Spielminute) durch andere User gecrashed wurde. Heißt, ich war gerade dran, beide Infos einzutragen, da wurde das neue Ergebnis als Halbzeit-oder Endergbenis gesetzt und meine Eingabe damit abgebrochen.
Hier wäre ein Flag sinnvoll die signalisiert, dass gerade jemand das neue Ergebnis pflegt.

**Comment by msiggi** [comment]
Ab sofort erscheint ein Hinweis, dass jemand gerade ein Tor bearbeitet. Das sollte zeigen, dass hier jemand am Werk ist und zeitgleiche unnötige Eingaben vermeiden.

![image](https://github.com/user-attachments/assets/517cd343-62cf-496e-ad87-772851568256)

Bitte testet es mal und gebt hier Feedback.

**Reply by xdream77** [reply]
Sieht gut aus für mich. 
Ist das zeitlich begrenzt? 
Ich habe die Bearbeitung eine Weile offen gelassen und dann ein paar Tabs gewechselt usw. und irgendwann ist die Nachricht verschwunden. 

Good Job 👍 

![Screenshot 2025-01-04](https://github.com/user-attachments/assets/7f3a0977-bba4-4b63-9738-5fe8405b017a)


PS: Du hast mit deinen Tests für dieses Feature in der BL1 updates in MQTT getriggert.

**Comment by msiggi** [comment]
Ja, wenn das "Tor Editieren"-Popup geöffnet wird und 20 Sekunden lang keine Änderung an einem der Eingabefelder erfolgt, wird der "TorWirdGeradeBearbeitet"-Status zurückgesetzt.

PS: Ja, kann sein ;-)

**Reply by xdream77** [reply]
Das war nicht so cool, weil der original Zustand nicht mit einer neuen **lastUpdateDateTime** versehen wurde. Das hat einiges zerschossen. (Hoffentlich nur bei mir). 

Gibt es eigentlich einen Discord Channel oder vergleichbares für OLDB? 
Da könnte man sich direkt miteinander austauschen :-) 

PS: OLDB ist ein richtig geiles Projekt :-)
